src/osx/cocoa/stdpaths.mm

```diff
@@ -143,9 +143,7 @@ ConfigFileConv WXUNUSED(conv)) const
 {
     // Shared libraries on OSX should be stored inside the
     // <Bundle.app>/Contents/Frameworks
-    wxFileName fn( GetExecutablePath() );
-    fn.RemoveLastDir();
-    return fn.GetPathWithSep() + "Frameworks/";
+    return wxCFStringRef::AsString((CFStringRef)[NSBundle mainBundle].privateFrameworksPath);
```

Comment:
Why do you need the cast here? Using `NSString*` directly should work too, i.e.
```suggestion
    return wxCFStringRef::AsString([NSBundle mainBundle].privateFrameworksPath);
```

Reply:
@vadz ,
There is another function that uses this cast, so I just copied it here.

Will check and if its not needed - will remove.

Do you want to fix the other function as well?

Reply:
@vadz ,
I removed the cast, but only for that function.